### ISSUE TITLE
Could not find objdump location

### ISSUE THREAD

**Issue by peterdk** [bugsnag/bugsnag-android-gradle-plugin#161]
### Expected behavior
Upload NDK symbols

### Observed behavior

```
Error attempting to calculate objdump location: Failed to find executable objdump at C:\android-sdk-windows\ndk-bundle\toolchains\x86_64-4.9\prebuilt\windows-x86_64\bin\x86_64-linux-android-objdump
Unable to upload NDK symbols: Could not find objdump location for x86_64
```

### Steps to reproduce
Export app as APK

### Version
classpath 'com.bugsnag:bugsnag-android-gradle-plugin:4.2.0' 
implementation 'com.bugsnag:bugsnag-android-ndk:4.14.0'
Gradle plugin 3.4.0
Gradle 5.1.1
Windows 10

### Additional information
I checked the path in the terminal, and the executable is executed when copying the mentioned path.

**Comment by snmaynard** [comment]
I imagine this is an issue with running on windows. Are you able to run against linux/osx in the interim? Also it would be useful to confirm it is related to the o/s rather than something else.

**Comment by peterdk** [comment]
I cloned the project on OSX, and it succeeded to upload the ndk symbols.
I had to change the sharedObjectPath to use '/' instead of '\\', but also that change did not make it work on Windows.

**Comment by snmaynard** [comment]
Ok - thanks @peterdk!

**Comment by tomlongridge** [comment]
This is fixed in #163, to be released shortly.
